modules/mobile/app/controllers/mobile/v0/disability_rating_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_dependency 'mobile/application_controller'
+
+module Mobile
+  module V0
+    class DisabilityRatingController < ApplicationController
+      def index
+        response = rating_proxy.for_user
+        render json: Mobile::V0::DisabilityRatingSerializer.new(response)
+      end
+
+      private
+
+      def rating_proxy
+        @rating_proxy ||= Mobile::V0::DisabilityRating::Proxy.new(@current_user)
```

Comment:
In Rails a new controller instance is created on each request. This means as long as a class' method isn't accessed twice we shouldn't need to memoize (@thing ||= ) it. Places where memoization helps would be when we make parallel calls.